bower.json

```diff
@@ -23,6 +23,8 @@
     "require-handlebars-plugin": "1.0.0",
     "aloha-editor": "oerpub/Aloha-Editor#master",
     "select2": "3.5.4",
+    "babel-polyfill": "^0.0.1",
+    "l20n": "https://github.com/tomekstach/l20n.js.git",
```

Comment:
@tomekstach : Could this forked repo be moved into the `openstax` or `katalysteducation` organization? GitHub supports redirects so you would not have to fix up this file (thank you for patiently rebasing all these times 😤  )